src/models/SqlOutputContentProvider.ts

```diff
@@ -150,10 +150,15 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
             });
         });
 
-        // add http post handler for setting the selection in the editor
-        this._service.addPostHandler(Interfaces.ContentType.EditorSelection, (req, res): void => {
+        // add http get handler for setting the selection in the editor
+        this._service.addHandler(Interfaces.ContentType.EditorSelection, (req, res): void => {
             let uri = req.query.uri;
-            let selection: ISelectionData = req.body;
+            let selection: ISelectionData = {
+                startLine: parseInt(req.query.startLine, 10),
```

Comment:
it is the base but it is not the default. There is no default, the radix for the conversion is a required input for some reason 🙄